packages/expo/src/Updates/Updates.ts

```diff
@@ -26,7 +29,10 @@ export async function checkForUpdateAsync(): Promise<Object> {
   return returnObj;
 }
 
-export async function fetchUpdateAsync({ eventListener }: any = {}): Promise<Object> {
+export async function fetchUpdateAsync({ eventListener }: any = {}): Promise<{
```

Comment:
```suggestion
export async function fetchUpdateAsync({ eventListener }: {eventListener: (updateEvent: UpdateEvent) => unknown} = {}): Promise<{
```

`UpdateEvent` (or equivalent) probably exists in this codebase already, but if not here's DT's: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a8ab3f0817c7e7a1be2cfc42f6ce8b4f0d9c008f/types/expo/index.d.ts#L3400-L3412